### ISSUE TITLE
An important question !!!

### ISSUE THREAD

**Issue by y78h11b09** [zylo117/Yet-Another-EfficientDet-Pytorch#444]
I review your code , and  have a  fusion that  is  different from the orginal paper, and furthermore yet exist some different between 
        self.p4_down_channel = nn.Sequential(
                Conv2dStaticSamePadding(conv_channels[1], num_channels, 1),
                nn.BatchNorm2d(num_channels, momentum=0.01, eps=1e-3),
            )
            self.p5_down_channel= nn.Sequential(
                Conv2dStaticSamePadding(conv_channels[2], num_channels, 1),
                nn.BatchNorm2d(num_channels, momentum=0.01, eps=1e-3),
            )
and 
        self.p4_down_channel_2 = nn.Sequential(
                Conv2dStaticSamePadding(conv_channels[1], num_channels, 1),
                nn.BatchNorm2d(num_channels, momentum=0.01, eps=1e-3),
            )
            self.p5_down_channel_2 = nn.Sequential(
                Conv2dStaticSamePadding(conv_channels[2], num_channels, 1),
                nn.BatchNorm2d(num_channels, momentum=0.01, eps=1e-3),
            )

why not use self.p4_down_channel  and self.pt_down_channel in where  is used self.p4_down_channel_2 and  self.p5_down_channel_2   ????

**Comment by zylo117** [comment]
duplicated.
https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/issues/412

**Comment by SakuraMaxuetao** [comment]
I trained 500 epochs, but the problem is still unsolved. My dataset comes from ：https://public.roboflow.ai/object-detection/chess-full
